addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,23 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
+msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38027"
+msgid "This option decodes frames for both eyes from video containing Multiview Video Coding (MVC) as typically found on 3D Blu-ray. Disabling may improve performance if you don't require 3D"
+msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38208"
+msgid "Enable Full HD HDMI modes for 3D"
```

Comment:
we always use "stereoscopic 3D", so we should do here as well (and also in the description ofc).